config/kubernetes/ug-big-data/OWNERS

```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - sig-big-data-leads
+  - ug-big-data-leads
```

Comment:
This also needs to be updated in OWNERS_ALIASES:

https://github.com/kubernetes/org/blob/0345f0fc6ad302cbb52bf072094e9416b79417f1/OWNERS_ALIASES#L28